programmers-coding-test/soongu/greedy/1이 될 때까지.js

```diff
@@ -0,0 +1,20 @@
+const rl = require("readline-sync");
+
+let [ n, k ] = rl.question("N과 K를 입력해주세요.\n>> ").split(',');
+
+let count = 0;
+console.log(n);
+while(true) {
+  if (n % k == 0) {
+    n = n / k; 
+    count++;
+  } else {
+    n--;
+    count++;
+  }
+  if (n === 1) {
+    break;
+  }
+}
```

Comment:
count는 무조건 해주는 것이므로 if, else 안에 모두 들어갈 필요가 없다.
한 줄로 짜는 것이 가독성에 좋아 보인다.
```suggestion
while(true) {
  if (n % k == 0) n = n / k; 
  else n--;
  
  count++;
  if (n === 1) break;
}
```